File: micro-service-combination/src/main/java/cn/plantlink/MicroServiceApplication.java
package cn.plantlink;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.Banner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;

/**
 * @Author colddew
 * @Date 2021-10-06
 */
@EnableFeignClients
@SpringBootApplication
public class MicroServiceApplication {

    private static final Logger logger = LoggerFactory.getLogger(MicroServiceApplication.class);

    public static void main(String[] args) throws Exception {

        SpringApplication application = new SpringApplication(MicroServiceApplication.class);
        application.setBannerMode(Banner.Mode.CONSOLE);
        application.run(args);

        logger.info("micro-service-combination is running...");
    }
}

File: micro-service-server2/src/main/java/cn/plantlink/MicroServiceApplication.kt
package cn.plantlink

import org.slf4j.LoggerFactory
import org.springframework.boot.Banner
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class MicroServiceApplication

private val logger = LoggerFactory.getLogger(MicroServiceApplication::class.java)

fun main(args: Array<String>) {
    runApplication<MicroServiceApplication>(*args) {
        setBannerMode(Banner.Mode.CONSOLE)
        logger.info("micro-service-server2 is running...")
    }
}